code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -27,3 +27,8 @@
 			I.has_embedded()
 
 	return 1
+
+/mob/living/carbon/can_be_grabbed(var/mob/grabber, var/obj/item/grab/grab)
+	. = ..()
+	if(.)
+		LAssailant = iscarbon(grabber) && grabber
```

Comment:
Maybe kill this var? Appears to be some slime bullshit.

Reply:
It's used by slime friend recognition, removing it would break that pretty hard.